packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -345,18 +364,9 @@
 .list-item.item__active .item-icon-text .title {
   margin-left: 40px;
 }
```

Comment:
I wonder if these styles shouldn't also be cleaned-up (as well as `:global(.body__menu-open) .list-item.item__active .item-icon-text .title` down below). Right now we end up with:
<img width="309" alt="Screenshot 2023-09-08 at 09 42 37" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/7409978d-a28e-4a14-828d-da670bfa4c7d">

Reply:
That little offset of the app label when hovered, is intentional. [Here](https://www.figma.com/proto/gC6Yz03ThScI6Mk23cjKDt/Main-Navi2023?page-id=0%3A1&node-id=402-53029&scaling=min-zoom&starting-point-node-id=402%3A53029&show-proto-sidebar=1&mode=design&t=GQgUVTutEie620dg-1) a little preview of that

Reply:
Thanks for adding the animations in Figma ❤️
The offset in the animation is a result of the icon's enlargement. Here, we simply include `margin-left: var(--spacing-m);`

Reply:
Yes, @kark is right. The behaviour will be addressed when transitions will be worked on. Thanks for noticing!
[c566aec](https://github.com/commercetools/merchant-center-application-kit/pull/3203/commits/c566aecfdc01a7d9d64a8cb5298298922f7d9a58)